chronix2grid/generation/renewable/solar_wind_utils.py

```diff
@@ -40,7 +45,7 @@ def compute_wind_series(prng, locations, Pmax, long_noise, medium_noise, short_n
     Nt_inter = int(params['T'] // params['dt'] + 1)
```

Comment:
Make method "get_seasonal_pattern" as for loads ?

Reply:
This is not a part of the code I modified, I will not change it right now.